src/sentry/runner/commands/devserver.py

```diff
@@ -292,6 +299,9 @@ def devserver(
     if settings.SENTRY_USE_RELAY:
         daemons += [_get_daemon("ingest")]
 
+        if occurrence_ingest:
+            daemons += [_get_daemon("occurrences")]
```

Comment:
I'd say that if we want to put this behind `SENTRY_USE_RELAY` it should be a setting. If we want to keep it as an argument we pass then I'd move this logic out from behind `SENTRY_USE_RELAY` since we're explicitly asking it to start.

Reply:
good call